static/js/components/SharePopup.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import React from "react"
+import onClickOutside from "react-onclickoutside"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from "react-share"
+import { FacebookIcon, TwitterIcon, LinkedinIcon } from "react-share"
+
+export class SharePopupHelper extends React.Component<*> {
+  input: ?HTMLInputElement
+
+  handleClickOutside = () => {
+    const { closePopup } = this.props
+    closePopup()
+  }
+
+  selectInputText = () => {
```

Comment:
This needs some `preventDefault()` magic so the page doesn't scroll when you click it.